src/download.cpp

```diff
@@ -206,6 +206,13 @@ void DownloadFile(const std::string& url, IDownloadSink *sink, int flags)
 
     if ( !filename_set )
     {
+        char ioptionurl[512];
```

Comment:
I used 512 as this is what is used a few lines above for the `url_path` buffer.
I'll change that to get it dynamically